--- thunar/thunar-window.c.t33	2026-07-14 07:37:13.520959074 +0000
+++ thunar/thunar-window.c	2026-07-14 07:37:13.557959226 +0000
@@ -7130,6 +7130,7 @@
     { "aurora-dark", "Aurora \xc2\xb7 Dark" },    { "aurora-light", "Aurora \xc2\xb7 Light" },
     { "porcelain-dark", "Porcelain \xc2\xb7 Dark" }, { "porcelain-light", "Porcelain \xc2\xb7 Light" },
     { "cobalt-dark", "Cobalt \xc2\xb7 Dark" },    { "cobalt-light", "Cobalt \xc2\xb7 Light" },
+    { "nova-dark", "NOVA \xc2\xb7 Dark" },        { "nova-light", "NOVA \xc2\xb7 Light" },
   };
   GtkWidget *item;
   GtkWidget *submenu;
