--- thunar/thunar-shortcuts-model.c.t36	2026-07-14 08:33:18.211065710 +0000
+++ thunar/thunar-shortcuts-model.c	2026-07-14 08:33:32.464146229 +0000
@@ -1030,7 +1030,7 @@
   /* add the devices heading */
   shortcut = g_slice_new0 (ThunarShortcut);
   shortcut->group = THUNAR_SHORTCUT_GROUP_DEVICES_HEADER;
-  shortcut->name = g_strdup (_("Devices"));
+  shortcut->name = g_strdup (_("DEVICES"));
   thunar_shortcuts_model_add_shortcut (model, shortcut);
 
   /* the filesystem entry */
@@ -1073,7 +1073,7 @@
   /* add the network heading */
   shortcut = g_slice_new0 (ThunarShortcut);
   shortcut->group = THUNAR_SHORTCUT_GROUP_NETWORK_HEADER;
-  shortcut->name = g_strdup (_("Network"));
+  shortcut->name = g_strdup (_("NETWORK"));
   thunar_shortcuts_model_add_shortcut (model, shortcut);
 
   /* append the browse network entry if it is supported */
@@ -1104,7 +1104,7 @@
   /* add the places heading */
   shortcut = g_slice_new0 (ThunarShortcut);
   shortcut->group = THUNAR_SHORTCUT_GROUP_PLACES_HEADER;
-  shortcut->name = g_strdup (_("Places"));
+  shortcut->name = g_strdup (_("PLACES"));
   thunar_shortcuts_model_add_shortcut (model, shortcut);
 
   /* get home path */
